[Example] Add EGES example

Wang-Yu-Qing opened the pull request:
## Description
<!-- Brief description. Refer to the related issues if existed.
It'll be great if relevant reviewers can be assigned as well.-->

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [ ] Code is well-documented
- [x] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

## Changes
<!-- You could use following template
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)
-->

BarclayII commented:
Could you remove the CSV files and put a URL in the README for downloading (as well as where to put them) instead?

Wang-Yu-Qing commented:
> Could you remove the CSV files and put a URL in the README for downloading (as well as where to put them) instead?

I've removed the CSV files and added a data link in the README.

jermainewang commented on examples/pytorch/eges/main.py:
```diff
@@ -0,0 +1,110 @@
+    
+
+    
+
```
Remove all these trailing newlines.

jermainewang commented on examples/pytorch/eges/model.py:
```diff
@@ -0,0 +1,59 @@
+        pass
+    
+
+
```
Same here

jermainewang commented on examples/pytorch/eges/utils.py:
```diff
@@ -0,0 +1,219 @@
+
+    
+           
+
```
same here

Wang-Yu-Qing replied:
Hi, I've removed all these trailing newlines.

jermainewang reviewed (approved):
Thanks for the contribution!
